### ISSUE TITLE
Fix rendering of function arguments of `aria-hidden`

### ISSUE THREAD

**Issue by davidruvolo51** [davidruvolo51/rheroicons#8]
Browsers do no recognize when an icon is hidden (i.e., `aria-hidden = "true"`). This is due to an error in the conversion script, The handling of the aria attribute hidden returns an underscore instead of a dash. This is a typo and should be fixed.

The conversion script should be adjusted so that it adds `aria-hidden` only if indicated. Browsers do not interpret `false` as of yet.

- [x] Change underscore to dash
- [x] new logic that adds `aria-hidden` only when true
- [x] test
